compressors/build.gradle.kts

```diff
@@ -0,0 +1,10 @@
+subprojects {
+  // https://github.com/gradle/gradle/issues/847
+  group = "io.opentelemetry.compressors"
+  val proj = this
+  plugins.withId("java") {
+    configure<BasePluginExtension> {
+      archivesName.set("opentelemetry-compressor-${proj.name}")
+    }
+  }
+}
```

Comment:
is this file needed? we don't seem to need parent module `runtime-attach`

Reply:
Runtime attach gets around this by fully qualifying the names of the nested modules. For example, it has:

```
/runtime-attach
  /runtime-attach
    ...
  /runtime-attach-core
```

While this file allows us to avoid the redundant naming and still produce artifacts with the correct names (i.e. `opentelemetry-compressor-zstd`:
```
/compressors
  /zstd
```

This is the pattern we use in `opentelemetry-java` (e.g. see [opentelemetry-java/exporters/build.gradle.kts](https://github.com/open-telemetry/opentelemetry-java/blob/main/exporters/build.gradle.kts)), but I don't have a strong opinion. We can accomplish the same thing either way.